### DISCUSSION TITLE
Discussions around enabling ZiraLink to support TCP/UDP ports

### DISCUSSION THREAD

**Discussion by saeedmaghdam** [saeedmaghdam/ZiraLink.Api#13] [Announcements]
Why does `ZiraLink` should support `TCP/UDP` ports?

**Comment by YAS-SIIN** [comment]
There are many opportunities in supporting TCP/UDP, which include : 
Sharing database - sharing RabbitMQ and any other message broker servers.
Of course, we should take care of accessibility. And we shall restrict users to some functional ports.

**Reply by YAS-SIIN** [reply]
What should we set a name for new feature in API and service and web page? Since, we had **"projects"** name for managing domains. 
I'm waiting for your response. I've chosen **"app-projects"**. If it is possible, give me a name to this new feature.

**Reply by saeedmaghdam** [reply]
We can either use an `app-sharing-project` or `app-project` for the new feature, it should be clear enough for a project that is aimed to share an app/use a shared app. I would choose `app-project` because it is a short and simple name.
